README.md

```diff
@@ -1,2 +1,72 @@
-# Hello-Microverse
- My goal here is to master all of the tools and best practices I learned about  and there is no complex code .
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
```

Comment:
- [ ] Kindly note that the `Getting Started` section of the `README` holds instructions on how users can clone and run your project successfully on their local machine and for them to be able to do that we need to include those instructions so they can do that. Kindly include the subheading below the `Getting Started` section as demonstrated in the [readme template](https://github.com/microverseinc/readme-template) provided in the project requirement.